src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m

```diff
@@ -22,6 +22,7 @@ - (nonnull instancetype)initWithMetalKitView:(nonnull MTKView *)mtkView
         mtkView.colorPixelFormat = MTLPixelFormatBGRA8Unorm;
         mtkView.depthStencilPixelFormat = MTLPixelFormatDepth32Float_Stencil8;
         mtkView.sampleCount = 1;
+        mtkView.framebufferOnly = false;
```

Comment:
Why is this needed? It may be interesting to detail a bit more in a comment.

Reply:
sure, it's easy (inside Xcode) to do F1 on the member to access the header docs - but it's not something GitHub does (at least right now).

Reply:
comment added